applications/giftless/templates/cert.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1
```

Comment:
You shouldn't need to create a separate cert resource. Just pick one of the two ingresses to own the public certificate and add the `cert-manager.io/cluster-issuer: "letsencrypt-dns"` annotation to it. See the squareone ingress.